vyper/premade_contracts.py

```diff
@@ -2,7 +2,9 @@
 
 erc20 = """
 class ERC20():
+    def name() -> bytes32: pass
     def symbol() -> bytes32: pass
+    def decimals() -> uint256: pass
```

Comment:
@BenKaufman10 please add tests fore the extra two functions.

Reply:
I think you can just add some asserts in test_initiat in test_on_chain_market_maker.py

Reply:
Alright, I added it there.